libs/ui-lib/lib/ocm/components/clusterWizard/ClusterWizardContextProvider.tsx

```diff
@@ -91,6 +91,7 @@ const ClusterWizardContextProvider = ({
   const [wizardStepIds, setWizardStepIds] = React.useState<ClusterWizardStepsType[]>();
   const [wizardPerPage, setWizardPerPage] = React.useState(10);
   const [customManifestsStep, setCustomManifestsStep] = React.useState<boolean>(false);
+  const [isDissconnectedMode, setIsDissconnectedMode] = React.useState<boolean>(false);
```

Comment:
isDissconnectedMode does not have to be stored in state once you will change the Context to accept it as prop.